hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/HederaConfig.java

```diff
@@ -30,32 +30,17 @@ public record HederaConfig(
         @ConfigProperty(value = "transaction.minValidDuration", defaultValue = "15") long transactionMinValidDuration,
         @ConfigProperty(value = "transaction.minValidityBufferSecs", defaultValue = "10")
                 int transactionMinValidityBufferSecs,
-        @ConfigProperty(value = "recordStream.recordFileVersion", defaultValue = "6") int recordStreamRecordFileVersion,
-        @ConfigProperty(value = "recordStream.signatureFileVersion", defaultValue = "6")
-                int recordStreamSignatureFileVersion,
         @ConfigProperty(value = "allowances.maxTransactionLimit", defaultValue = "20")
                 int allowancesMaxTransactionLimit,
         @ConfigProperty(value = "allowances.maxAccountLimit", defaultValue = "100") int allowancesMaxAccountLimit,
         @ConfigProperty(value = "allowances.isEnabled", defaultValue = "true") boolean allowancesIsEnabled,
-        @ConfigProperty(value = "recordStream.enableTraceabilityMigration", defaultValue = "true")
-                boolean recordStreamEnableTraceabilityMigration,
-        @ConfigProperty(value = "recordStream.compressFilesOnCreation", defaultValue = "true")
-                boolean recordStreamCompressFilesOnCreation,
         @ConfigProperty(defaultValue = "data/onboard/exportedAccount.txt") String accountsExportPath,
         @ConfigProperty(defaultValue = "false") boolean exportAccountsOnStartup,
         @ConfigProperty(value = "prefetch.queueCapacity", defaultValue = "70000") int prefetchQueueCapacity,
         @ConfigProperty(value = "prefetch.threadPoolSize", defaultValue = "4") int prefetchThreadPoolSize,
         @ConfigProperty(value = "prefetch.codeCacheTtlSecs", defaultValue = "600") int prefetchCodeCacheTtlSecs,
         // @ConfigProperty(value = "profiles.active", defaultValue = "PROD") Profile profilesActive,
         @ConfigProperty(value = "profiles.active", defaultValue = "PROD") String activeProfile,
-        @ConfigProperty(value = "recordStream.isEnabled", defaultValue = "true") boolean recordStreamIsEnabled,
-        @ConfigProperty(value = "recordStream.logDir", defaultValue = "/opt/hgcapp/recordStreams")
```

Comment:
Are we removing these from `HederaConfig` for separating recordStream configs ? I thought, they are put into config based on the property names that started with `hedera. ....`.
Will we be changing the name that start with `hedera.` for these ?

Reply:
It works the same -- they're still prefixed with "hedera". See BlockRecordStreamConfig